data/frame_type.go

```diff
@@ -41,6 +41,20 @@ const (
 	// This structure is typically part of a list of frames with the same structure
 	FrameTypeTimeSeriesMany = "timeseries-many"
 
+	// FrameTypeLabeledTimeValues has at least three fields:
+	// field[0]:
+	//  * type: string field
+	//  * name: "labels"
+	//  * value: JSON serialized data.Labels structures
+	// field[1]:
+	//  * type: time
+	//  * ascending or decending values within each label set.
+	// field[2..n]:
+	//  * type: any
+	// NOTE: no field [0..n] should have a populated "labels" attribute
+	// NOTE: this format is similar to influx line protocol, except that the field set is consistent
+	FrameTypeLabeledTimeValues = "labeled-time-values"
```

Comment:
@kylebrandt -- any better?  maybe "labeled-timeseries-values" 

It is timeseries, but it may be strings and or numbers so 🤷  -- ie... influxdb